README.md

```diff
@@ -14,6 +14,21 @@ The project was *initially* forked from the  upstream [mozilla-services/go-cose]
 
 We thank the [Mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraison-go-cose] project.
 
+## Community
+
+The [veraison/go-cose](https://github.com/veraison/go-cose) project is an open source community effort.
+
+You can reach the go-cose community via::
+
+- [Distribution Group](https://groups.io/g/go-cose-community)
+- Bi-weekly meetings: 8:00-9:00 Pacific
+  - [Zoom meeting link](https://us02web.zoom.us/j/81054434992?pwd=YjNBU21seU5VcGdtVXY3VHVjS251Zz09)
+  - [Calendar ics link](https://zoom.us/meeting/tZUtcu2srT8jE9YFubXn-lC9upuwUiiev52G/ics)
+- [Meeting Notes](https://hackmd.io/9dpXsrBxTrefxrNaTgq12Q)
+- [Meeting Recordings](https://www.youtube.com/@go-cose-community3000)
+
+Participation in the go-cose community is governed by the [Code of Conduct](CODE_OF_CONDUCT.md) and [veraison/community governance](https://github.com/veraison/community/blob/main/GOVERNANCE.md)
```

Comment:
I would suggest pulling this line in favor of https://github.com/veraison/go-cose/pull/128 which moves the CODE_OF_CONDUCT.MD to be inherited from the .github repo in this org rather than using the repo specific version

Reply:
Line updated with the file removed.